src/CMakeLists.txt

```diff
@@ -876,11 +868,10 @@ endif(WITH_LIBCEPHFS)
 
 add_subdirectory(journal)
 
-if(LINUX)
-  add_library(krbd_objs OBJECT krbd.cc)
-endif()
-
 if(${WITH_RBD})
```

Comment:
@bassamtabbara 
Yup, that is the normal invocation in the project, yours would work also in this specific case but still without `${}` is preferred. 
But if you want to know, loks up variable in the cmake manual